task/deprecated-image-check/0.1/deprecated-image-check.yaml

```diff
@@ -35,7 +35,8 @@ spec:
       image: registry.access.redhat.com/ubi8/ubi
       script: |
         #!/usr/bin/env bash
-        IFS=$'\n' read -d'' -r -a IMAGE_ARRAY < <(echo -n $BASE_IMAGES_DIGESTS | sed 's/\\n/\'$'\n''/g' && printf '\0')
+BASE_IMAGES_DIGESTS=$(echo $(params.BASE_IMAGES_DIGESTS) | sed 's/\\n/\'$'\n''/g' && printf '\0')
```

Comment:
Sorry, I just wanted to leave a comment here, but I committed it accidentally.

Reply:
no Problem